keyboards/handwired/d48/keymaps/default/rules.mk

```diff
@@ -0,0 +1,4 @@
+
+USER_NAME = anderson
```

Comment:
Userspace stuff should not be in the default keymap.

Reply:
Done. Added default keymap & moved existing stuff into extra keymap.